eng/vmr-sync.sh

```diff
@@ -259,15 +259,15 @@ fi
 
 # Synchronize the VMR
 
+# TODO https://github.com/dotnet/arcade-services/issues/3024: Re-enable TPN synchronization once the tooling is fixed
```

Comment:
This appears to be fixed so why not incorporate that change here so the TPN sync doesn't need to be disabled?

Reply:
Ah yeah, it just built now, I will bump it here as well